.github/workflows/deploy-vue-storefront-cloud.yml

```diff
@@ -46,64 +46,27 @@ jobs:
           NPM_PASS: ${{ secrets.NPM_PASS }}
           NPM_USER: ${{ secrets.NPM_USER }}
           NPM_REGISTRY: "https://registrynpm.storefrontcloud.io"
-
-  deploy:
+  
+  deployment-app:
```

Comment:
Issue: this should be "deploy" because our cloud.service ciTemplate will crash...

Reply:
Done: a94efad32eab750005e153d3bc954223758603aa